<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-08T18:53:57 EDT" name="Default suite" finished-at="2023-06-08T18:54:13 EDT" duration-ms="16626">
    <groups>
    </groups>
    <test started-at="2023-06-08T18:53:57 EDT" name="Default test" finished-at="2023-06-08T18:54:13 EDT" duration-ms="16626">
      <class name="ca.amazon.login.SearchTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:ca.amazon.login.SearchTest@738dc9b]" started-at="2023-06-08T18:53:57 EDT" name="setUp" finished-at="2023-06-08T18:54:03 EDT" duration-ms="6072" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="searchProductPartialText()[pri:1, instance:ca.amazon.login.SearchTest@738dc9b]" started-at="2023-06-08T18:54:03 EDT" name="searchProductPartialText" finished-at="2023-06-08T18:54:08 EDT" duration-ms="5438" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Search results page is not displayed.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Search results page is not displayed.
at org.testng.Assert.fail(Assert.java:99)
at ca.amazon.login.SearchTest.searchProductPartialText(SearchTest.java:85)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchProductPartialText -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:ca.amazon.login.SearchTest@738dc9b]" started-at="2023-06-08T18:54:08 EDT" name="tearDown" finished-at="2023-06-08T18:54:13 EDT" duration-ms="5065" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- ca.amazon.login.SearchTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
